rin_serin/week04/mission02/src/pages/MovieIdPage.jsx

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import { useLocation, useParams } from "react-router-dom";
+import useCustomFetch from "../hooks/useCustomFetch";
+import { useState } from "react";
+
+const All= styled.div`
+    width: 100%;
+`
+const Banner=styled.div`
+    background-image: url(${props=>props.backImg});
+    padding: 16px 20px;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 320px;
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+`
+const Title=styled.h2`
+    font-size: 20px;
+`
+const Rate=styled.p`
+    font-size: 16px;
+`
+const Subtitle=styled.p`
+    font-size: 16px;
+`
+const Actors=styled.div`
+    width: 100%;
+    padding: 16px 20px;
+`
+const People=styled.div`
+    display: flex;
+`
+const Person=styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`
+
+const MovieIdPage = (props) => {
+    const location = useLocation();
+
+    const movieId = useParams(); // URL 파라미터로 movieId 가져오기
+    console.log(movieId);
```

Comment:
```suggestion
    const { movieId }  = useParams(); // URL 파라미터로 movieId 가져오기
```

console로 객체 들어오시는 거 확인해주시고, 객체에 movieId라는 Key 값과 value가 들어오는거 확인되시면 위에처럼 구조분해할당으로 가져와주세요!